File: euler_6.py
def sumofsquare(numberList):
	result=sum([x**2 for x in numberList])
	return result

def squareofsum(numberList):
	import math
	result=math.pow(sum(numberList), 2)
	return result
	
#print(sumofsquare(range(1,11))
#print(squareofsum(range(1,11))

r = sumofsquare(range(1,101))
print(r)

r2 = int(squareofsum(range(1,101)))
print(r)

print(r2-r)

File: need4spd/euler_6.rb
def sumofsquare(numberList)
	result=0
	numberList.each do |number|
		result=result+(number**2)
	end
	
	return result
end

def squareofsum(numberList)
	result=0
	numberList.each do |number|
		result=result+number
	end
	
	result=result**2
	
	return result
end


r=sumofsquare(Array(1..100))
r2=squareofsum(Array(1..100))

puts r
puts r2
puts r2-r